### ISSUE TITLE
[ISSUE]: script closes and opens endlessly when starting from explorer or non admin CMD.EXE instance

### ISSUE THREAD

**Issue by israpps** [GDX-X/PFS-BatchKit-Manager#26]
no further explanation needed

**Comment by GDX-X** [comment]
Try to download again

**Comment by israpps** [comment]
> Try to download again

Happens on latest too

**Comment by GDX-X** [comment]
The problem seems to come from your side for the moment I tried on two computers + a virtual machine I do not have this bug

**Comment by seascape** [comment]
This happened for me just now (first time user). I turned ECHO on and it seemed to be exiting then restarting at the point it tried to call the vbs admin script. (To ensure PFS Batchkit Manager had admin I manually ran its .bat file as admin at all times.) When the script fell into its open/close loop the only remedy I found was logging out of Windows. I only got around the bad loop by deleting the parts of the script before the main menu, after which it seemed to function normally.

I was launching the script from a third-party Windows Explorer alternative called XYplorer. If I go back to this I can try running an unedited copy of the script from standard Windows Explorer.

Just to note I also ran into an error where PFS Batchkit Manager could not successfully install the Dokan driver. Unclear if it's due to the part of the script I deleted or not.
